test_scripts/API/Restructuring_OnResetTimeout/004_BC_OnResetTimeout_HMI_did_not_respond_notification_without_resetPeriod.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(without resetPeriod) to SDL in 2 seconds after HMI request receiving
```

Comment:
it says `in 2 seconds`, but I see `notificationTime = 6000`. the description does not match the code.

which one is correct? the description? or the code?